content/en/docs/examples/microservices-istio/add-istio/index.md

```diff
@@ -0,0 +1,139 @@
+    Istio-enabled pods, transparently and incrementally,  performing what is
+    called a
+    [rolling update](https://kubernetes.io/docs/tutorials/kubernetes-basics/update-intro/).
+    Kubernetes terminated an old pod only when a new pod started to run, and it
+    transparently switched the traffic to the new pods, one by one. (To be more
+    precise, it did not terminate more than one pod before a new pod was
+    started.) All this was done to prevent disruption of your application, so it
+    continued to work during the injection of Istio.
+
+1.  Check the logs of the Istio sidecar of `productpage`:
+
+    {{< text bash >}}
+    $ kubectl logs -l app=productpage -c istio-proxy | grep GET
+    ...
+    [2019-02-15T09:06:04.079Z] "GET /details/0 HTTP/1.1" 200 - 0 178 5 3 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Safari/605.1.15" "18710783-58a1-9e5f-992c-9ceff05b74c5" "details:9080" "172.30.230.51:9080" outbound|9080||details.tutorial.svc.cluster.local - 172.21.109.216:9080 172.30.146.104:58698 -
+    [2019-02-15T09:06:04.088Z] "GET /reviews/0 HTTP/1.1" 200 - 0 379 22 22 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Safari/605.1.15" "18710783-58a1-9e5f-992c-9ceff05b74c5" "reviews:9080" "172.30.230.27:9080" outbound|9080||reviews.tutorial.svc.cluster.local - 172.21.185.48:9080 172.30.146.104:41442 -
+    [2019-02-15T09:06:04.053Z] "GET /productpage HTTP/1.1" 200 - 0 5723 90 83 "10.127.220.66" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Safari/605.1.15" "18710783-58a1-9e5f-992c-9ceff05b74c5" "tutorial.bookinfo.com" "127.0.0.1:9080" inbound|9080|http|productpage.tutorial.svc.cluster.local - 172.30.146.104:9080 10.127.220.66:0 -
+    {{< /text >}}
+
+1.  Output the name of your namespace. You will need it to recognize your
+    microservices in the Istio dashboard:
+
+    {{< text bash >}}
+    $ echo $(kubectl config view -o jsonpath="{.contexts[?(@.name == \"$(kubectl config current-context)\")].context.namespace}")
+    tutorial
+    {{< /text >}}
+
+1.  Check the Istio dashboard, access
+    [http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard](http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard).
+    (The `my-istio-dashboard.io` URL should be in your /etc/hosts file, you set
+    it
+    [previously](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)).
+
+    In the top left drop-down menu, select _Istio Mesh Dashboard_.
+
+    {{< image width="80%"
+        link="dashboard-select-dashboard.png"
+        caption="Select Istio Mesh Dashboard from the top left drop-down menu"
+        >}}
+
+    Notice the `productpage` service from your namespace, it's name should be
+    `productpage.<your namespace>.svc.cluster.local`.
```

Comment:
having the user explicitly provide a namespace may make this simpler

Reply:
Namespace should be named tutorial, unless the user chose a different one earlier in the tutorial where it's mentioned.